CMakeLists.txt

```diff
@@ -967,17 +967,13 @@ if(HPX_WITH_LIBCDS)
   hpx_option(
     HPX_WITH_LIBCDS_GIT_REPOSITORY STRING
     "Define the LibCDS git repository to use."
-    https://github.com/STEllAR-GROUP/libcds CATEGORY "Thread Manager" ADVANCED
+    https://github.com/khizmax/libcds.git CATEGORY "Thread Manager" ADVANCED
   )
   hpx_option(
-    HPX_WITH_LIBCDS_GIT_TAG STRING "Define the LibCDS git tag to use." hpx-1.5
-    CATEGORY "Thread Manager" ADVANCED
+    HPX_WITH_LIBCDS_GIT_TAG STRING "Define the LibCDS git tag to use."
+    master CATEGORY "Thread Manager" ADVANCED
```

Comment:
I would like to see a tag here instead of a reference to an ever moving branch.

Reply:
It needs efforts from LibCDS, is it correct?